config/default.yaml

```diff
@@ -105,9 +109,9 @@ dspace::java_version : '7'
 # Always build/compile DSpace using "vagrant.properties"
 # instead of the normal "build.properties" file
 # IF you want to exclude particular webapps from building & installing, you can do so like:
-# mvn_params: '-Denv=vagrant -P!dspace-lni,!dspace-rdf'
+# mvn_params: '-P-dspace-lni,-dspace-rdf'
 # (This setting is only used during a 'vagrant up')
-mvn_params        : '-Denv=vagrant'
+mvn_params        : ''
```

Comment:
For better performance / quicker builds, we may want to exclude 'dspace-rdf' by default. It requires installing additional software (a triplestore) anyways, which we don't provide by default in vagrant-dspace (see https://wiki.duraspace.org/display/DSDOC5x/Linked+%28Open%29+Data). So, until we are also installing a triplestore, we likely should exclude it by default for now.